bitcoin-core-dev-tech/2015-02/jeremy-allaire-circle.es.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Círculo 
```

Comment:
I know this is a literal translation of the original but maybe we can add a little more description to this